README.adoc

```diff
@@ -248,18 +255,18 @@ https://www.ibm.com/support/knowledgecenter/was_beta/com.ibm.websphere.wlp.nd.mu
 
 == Enabling distributed tracing
 
-While the OpenTracing feature (`opentracing-1.0`) found inside Open Liberty enables tracing of all JAX-RS methods by default,
-MicroProfile OpenTracing allows you to control and customize these traces. Using the `@Traced` annotation
-you can enable and disable tracing of particular methods. A custom `Tracer` object can also be injected
-for creating and customizing spans.
+The OpenTracing feature (`opentracing-1.0`) found inside Open Liberty enables tracing of all JAX-RS
+methods by default, but you can use MicroProfile OpenTracing to control and customize these traces.
+Use the `@Traced` annotation to enable and disable tracing of particular methods. You can also inject
+a custom `Tracer` object to create and customize spans.
```

Comment:
Is there any way we can remove the parenthesis without removing the content inside of it? Parenthesis indicate that the content is less important.

Reply:
What if we just remove them without adding anything else? ie. "The OpenTracing feature opentracing-1.0 found inside ..."

Reply:
Usually we write the name of the feature/URL/whatever it is first, so it would be..."The `opentracing-1.0` OpenTracing feature..." But, yes, otherwise I think it's good. :)

Reply:
Ok will do that